src/Fetch.ml

```diff
@@ -379,6 +381,48 @@ module Response = struct
   external clone : t = "clone" [@@bs.send.pipe: t]
 end
 
+module FormData = struct 
+  module EntryValue = struct
+    type t
+
+    let classify t : [> `String of string | `File of file] =
+      if Js.typeof t = "string" then `String (Obj.magic t)
+      else `File (Obj.magic t)
```

Comment:
This still doesn't constrain the input argument, so you could for example do `classify 42` which would return an invalid `` `File`` value back.
```suggestion
    let classify (t: t) : [> `String of string | `File of file] =
      if Js.typeof t = "string" then `String (Obj.magic t)
      else `File (Obj.magic t)
```

Reply:
Will change that.